src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Sql/Sprocs/UpdateWorkitemStatus.sql

```diff
@@ -0,0 +1,44 @@
+﻿/*************************************************************
+    Stored procedure for updating a workitem status.
+**************************************************************/
+--
+-- STORED PROCEDURE
+--     UpdateWorkitemStatus
+--
+-- DESCRIPTION
+--     Updates a workitem status.
+--
+-- PARAMETERS
+--     @partitionKey
+--         * The system identifier of the data partition.
+--     @workitemKey
+--         * The workitem key.
+--     @status
+--         * Initial status of the workitem status, Either 0(Creating) or 1(Created)
```

Comment:
This should say `New status of the workitem, ...`

Reply:
Added this in the next PR